frontend/packages/dev-console/src/utils/application-utils.ts

```diff
@@ -456,11 +458,11 @@ export const cleanUpWorkload = (
       deleteRequest(modelFor(resource.kind), resource);
       batchDeleteRequests(deleteModels, resource);
       deleteRequest(ImageStreamModel, resource); // delete imageStream
-      webhooksAvailable = true;
+      isBuildConfigPresent ? (webhooksAvailable = true) : (webhooksAvailable = false);
```

Comment:
`webhooksAvailable = isBuildConfigPresent` ?

Reply:
@sahil143 This may not be true always i.e if buildConfig is present then it'll have `webhooks` as even in UI we have option to select webhooks. So we may need to handle it

Reply:
@sahil143  @invincibleJai so `generic` webhook is always going to be there since we are adding it always so here we can set `webhooksAvailable` based on `builldConfig` is present or not. Below at line no. 470 inside the if block we can check if some other webhook also exist using `checkIfTriggerExists` from `edit-application-utils` and if it exists then based on that add that hook into the webhooks array.

Reply:
+1 for `webhooksAvailable = isBuildConfigPresent;`